tests/e2e/e2e_upgrade_handler_test.go

```diff
@@ -98,6 +99,26 @@ func (suite *IntegrationTestSuite) TestUpgradeParams_Consensus() {
 	suite.Require().Equal(expectedParams, *paramsAfter.Params, "x/consensus params after upgrade should be as expected")
 }
 
+func (suite *IntegrationTestSuite) TestUpgradeParams_CDP_Interval() {
+	suite.SkipIfUpgradeDisabled()
+
+	beforeUpgradeCtx := suite.Kava.Grpc.CtxAtHeight(suite.UpgradeHeight - 1)
+	afterUpgradeCtx := suite.Kava.Grpc.CtxAtHeight(suite.UpgradeHeight)
+
+	grpcClient := suite.Kava.Grpc
+
+	paramsBefore, err := grpcClient.Query.Cdp.Params(beforeUpgradeCtx, &cdptypes.QueryParamsRequest{})
+	suite.Require().NoError(err)
+	paramsAfter, err := grpcClient.Query.Cdp.Params(afterUpgradeCtx, &cdptypes.QueryParamsRequest{})
+	suite.Require().NoError(err)
+
+	expectedParams := paramsBefore.Params
+	expectedParams.LiquidationBlockInterval = int64(100)
+
+	suite.Require().Equal(expectedParams, paramsAfter.Params,
```

Comment:
Tested that this failed when modifying another parameter beyond the block interval